### PR TITLE
Support fetch step logs

### PR DESCRIPTION
**Pull request by tonyyang-svail**
Part of #1367.

Next Step: https://github.com/sql-machine-learning/sqlflow/blob/56602cabdd26806912953a9e2254eb6207489fee/pkg/argo/fetch.go#L36

### CODE REVIEW

File: pkg/argo/submit.go [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+	output, err := cmd.CombinedOutput()
+	if err != nil {
+		return "", fmt.Errorf("submit Argo YAML error: %v, output: %s", err, string(output))
+	}
+
+	workflowID, err := getWorkflowID(string(output))
+	if err != nil {
+		return "", err
+	}
+	return workflowID, err
```
Comment (weiguoz) [inline pkg/argo/submit.go:35]: just `return getWorkflowID(string(output))`

File: pkg/proto/sqlflow.proto [hunk 2]
```diff
@@ -67,8 +67,9 @@ message FetchResponse {
         SUCCEEDED = 2;
         FAILED = 3;
         ERROR = 4;
+        SKIPPED = 5;
```
Comment (Yancey0623) [inline pkg/proto/sqlflow.proto:70]: Do we really need the `SKIPPED` phase, I don't know what does the client do for the skipped phase.

File: pkg/argo/submit.go (hunk 1 above)
Comment (tonyyang-svail) [reply pkg/argo/submit.go:35]: Good point.

**Review by Yancey0623 — APPROVED** [review]
LGTM  with a tiny comment, please confirm it.

File: pkg/proto/sqlflow.proto (hunk 2 above)
Comment (tonyyang-svail) [reply pkg/proto/sqlflow.proto:70]: According to the Argo source code, the client should consider SKIPPED as the completion phase. I suspect skip is only used for nodes (for example the skipped if branch), never for the workflow. But I feel it is safer to follow the Argo source.

https://github.com/argoproj/argo/blob/99db30d67b42cbd9c7fa35bbdd35a57040c2f222/pkg/apis/workflow/v1alpha1/workflow_types.go#L841-L846

**Pull request merged**